apps/dashboard/src/components/workflow-status.tsx

```diff
@@ -32,7 +40,7 @@ export const WorkflowStatus = (props: WorkflowStatusProps) => {
   const text = statusRenderData[status].text;
 
   return (
-    <Badge variant={badgeVariant}>
+    <Badge variant={badgeVariant} className="border-none py-1">
```

Comment:
Why the extra styles here?

Reply:
after the rebase it doesnt needs the py-1, removing it. but it needs the "border-none" for the inactive state